src/main/java/com/lambdaschool/oktafoundation/OktaFoundationApplication.java

```diff
@@ -13,6 +13,7 @@
 @EnableJpaAuditing
 @SpringBootApplication
 //@PropertySource("file:/E:/Coding/configs/oktafoundationconf.properties")
+@PropertySource("file:/C:/Users/georg/ev/oktafoundationconf.properties")
```

Comment:
I think it will be better to not have line 16 on the main branch as it is used just for your local machine.